packages/control-property-editor/src/panels/properties/StringEditor.tsx

```diff
@@ -38,7 +38,8 @@ export function StringEditor(propertyInputProps: PropertyInputProps): ReactEleme
     } = propertyInputProps;
     const [val, setValue] = useState(value);
     const icons = useSelector<RootState, IconDetails[]>((state) => state.icons);
-
+    const scenario = useSelector<RootState, Scenario>((state) => state.scenario);
+    const isAdpProject = scenario === 'ADAPTATION_PROJECT';
     useEffect(() => {
```

Comment:
Scenarios can be moved in some file for constants because they are used already in other files e.g App.tsx

Reply:
This change has already been done in [1553](https://github.com/SAP/open-ux-tools/pull/1553) and can be integrated here after it is merged